### PR TITLE
Add cookie consent banner to base.mako [PLAT-904]

### PR DESCRIPTION
**Pull request by erinspace**
cosio_cookiebanner

Places a banner on the bottom of all pages on cos.io with links to the privacy policy and cookie information.

When clicking Accept, sets a cookie to remove the banner that expires in 30 days.

Also apparently my editor removed a lot of trailing spaces in `style.css` - this wasn't intentional but probably for the best, lemmie know and I can put them back.

## Ticket
https://openscience.atlassian.net/browse/PLAT-904

### CODE REVIEW

**Comment by erinspace** [comment]
Note -- banner is now same color as [OSF's banner](https://github.com/CenterForOpenScience/osf.io/pull/8524)

screen shot 2018-07-10 at 1 29 05 pm

**Comment by erinspace** [comment]
link colors upadted

screen shot 2018-07-12 at 1 57 49 pm

File: cos/static/scripts/cookie-banner.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+$(document).ready(function () {
+    var cookieBannerElem = $('#cookieBanner');
+    var cookieAcceptedKey = 'cos_cookieconsent';
+    var daysToExpire = 30;
+
+    var $cookieBanner = cookieBannerElem.on('closed.bs.alert', function() {
+        var d = new Date();
+        d.setTime(d.getTime() + (daysToExpire*24*60*60*1000));
+        var expires = "expires="+ d.toUTCString();
+        document.cookie = cookieAcceptedKey + "=;" + expires + ";path=/";
```
Comment (caseyrollins) [inline cos/static/scripts/cookie-banner.js]: Does a cookie not have to have a value?

File: cos/static/scripts/cookie-banner.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// Cookie Banner
+$(document).ready(function () {
+    var cookieBannerElem = $('#cookieBanner');
+    var cookieAcceptedKey = 'cos_cookieconsent';
+    var daysToExpire = 30;
+
+    var $cookieBanner = cookieBannerElem.on('closed.bs.alert', function() {
```
Comment (caseyrollins) [inline cos/static/scripts/cookie-banner.js]: Nit: doesn't look like this var is used anywhere

**Review by caseyrollins — APPROVED** [review]
LGTM. Just a question and non blocking comment.

File: cos/static/scripts/cookie-banner.js (hunk 1 above)
Comment (erinspace) [reply cos/static/scripts/cookie-banner.js]: I didn't think so, but couldn't hurt to set one cause that's definitely clearer, and matches what the OSF is doing. I'll change it to 1!

**Pull request merged**